Gemfile

```diff
@@ -0,0 +1,13 @@
+source 'https://rubygems.org'
+
+ruby '3.1.1'
```

Comment:
Thought: Is this version of ruby really necessary? It isn't. would it then be better not to require it?

I understand that Gemfile is there to help another user have the same background software as the developer to ensure that the program also works for them. You used 3.1.1. so you are telling other users to do the same.

On the other hand, installing the latest ruby takes time (in fact for some reason, it's taking forever for me - I am running 3.0.0 and have 3.1.0 installed too but not 3.1.1), so if it's not required, would it be better to allow users to run your program on an earlier version of ruby?

I'm not sure how you would decide which earlier version you would use.